code/traffic-register/traffic-register.css

```diff
@@ -322,6 +381,11 @@ a.ang-link:link {text-decoration: none;}
   box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
 #kn-scene_417 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
 
+/*Fix Reg Errors*/
```

Comment:
@ChrispinP what does this refer to: Fixing what errors? Like an error message banner?

Reply:
This is the page name